examples/website/geojson/app.js

```diff
@@ -11,56 +11,75 @@ const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
 const DATA_URL =
   'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/geojson/vancouver-blocks.json'; // eslint-disable-line
 
-const colorScale = r => [r * 255, 140, 200 * (1 - r)];
+const LIGHT_SETTINGS = {
+  lightsPosition: [-125, 50.5, 5000, -122.8, 48.5, 8000],
+  ambientRatio: 0.2,
+  diffuseRatio: 0.5,
+  specularRatio: 0.3,
+  lightsStrength: [1.0, 0.0, 2.0, 0.0],
+  numberOfLights: 2
+};
 
-class Root extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      viewport: {
-        ...DeckGLOverlay.defaultViewport,
-        width: 500,
-        height: 500
-      },
-      data: null
-    };
-
-    fetch(DATA_URL)
-      .then(resp => resp.json())
-      .then(data => this.setState({data}));
-  }
+const INITIAL_VIEW_STATE = {
+  latitude: 49.254,
+  longitude: -123.13,
+  zoom: 11,
+  maxZoom: 16,
+  pitch: 45,
+  bearing: 0
+};
 
-  componentDidMount() {
-    window.addEventListener('resize', this._resize.bind(this));
-    this._resize();
-  }
+const DEFAULT_COLOR_SCALE = r => [r * 255, 140, 200 * (1 - r)];
 
-  _resize() {
-    this._onViewportChange({
-      width: window.innerWidth,
-      height: window.innerHeight
-    });
+export class App extends Component {
+  static get defaultViewport() {
```

Comment:
Is this still needed?

Reply:
I believe that there is a viewport transition functionality in the "DemoLauncher". I think it is the one piece of existing functionality that I didn't maintain. 

I will take another look, assuming we need to keep this for now.

Reply:
I don't think we have to keep the transition effect.